components/FeatureHighlight/examples/FeatureHighlightThemerExample.m

```diff
@@ -77,6 +54,19 @@ - (void)didTapButton:(id)sender {
                                                               completion:nil];
   [MDCFeatureHighlightAccessibilityMutator mutate:vc];
 
+  // Typography theming.
+  MDCTypographyScheme *typographyScheme = [[MDCTypographyScheme alloc] init];
+  typographyScheme.headline6 = [UIFont fontWithName:@"Zapfino" size:14.0];
+  typographyScheme.body2 = [UIFont fontWithName:@"Chalkduster" size:12.0];
+  [MDCFeatureHighlightTypographyThemer applyTypographyScheme:typographyScheme
+                            toFeatureHighlightViewController:vc];
+
+  // Color Theming
+  MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] init];
+  colorScheme.primaryColor = [UIColor purpleColor];
```

Comment:
The default theme is already purple - is this line necessary?

Reply:
I thought primary is blue? I just kept the logic that existed

Reply:
Hmm the more I look at this example the more I think it may not be necessary - the typical use example should show how to theme a feature highlight with our schemes. If/when the app's global theme changes, then it will be apparent how it affects feature highlight in the typical example. We might be able to delete this example entirely.

Reply:
Hmm this examples lives only in Dragon, do you want me to still get rid of it?

Reply:
Done.